.github/ISSUE_TEMPLATE/new-member.yml

```diff
@@ -22,6 +22,13 @@ body:
       label: Which project(s) are you looking to contribute to?
     validations:
       required: false
+  - type: checkboxes
+    id: tangible-benefits
+    attributes:
+      label: Tangible Benefits
+      options:
+        - label: Does your project provide tangible benefits to the Django Commons community?
```

Comment:
I'd also be good dropping the commons part of this question.

Reply:
removed the ‘commons’ part of the question